pkg/components/components.go

```diff
@@ -14,19 +14,21 @@ import (
 const Name = "cluster-network-addons-operator"
 
 const (
-	MultusImageDefault         = "docker.io/nfvpe/multus:latest"
-	LinuxBridgeCniImageDefault = "quay.io/kubevirt/cni-default-plugins:latest"
-	SriovDpImageDefault        = "quay.io/booxter/sriov-device-plugin:latest"
-	SriovCniImageDefault       = "docker.io/nfvpe/sriov-cni:latest"
-	KubeMacPoolImageDefault    = "quay.io/schseba/mac-controller:latest"
+	MultusImageDefault            = "docker.io/nfvpe/multus:latest"
+	LinuxBridgeCniImageDefault    = "quay.io/kubevirt/cni-default-plugins:latest"
+	LinuxBridgeMarkerImageDefault = "quay.io/kubevirt/bridge-marker:latest"
```

Comment:
please release a tagged image and pin it here

Reply:
Added 0.1.0